Add paragraph describing ToS URL usage.

cpu opened the pull request:
This commit adds a line to the integration guide's "Plan for Change" section to explicitly mention that you should *not* hard-code the ToS URL since it may change, and instead should follow the ACME spec's recommendation for getting the current ToS URL from the new-reg response.

rolandshoemaker commented on docs/integration-guide.md:
```diff
@@ -15,6 +15,8 @@ Both Let's Encrypt and the Web PKI will continue to evolve over time.  You shoul
 
 In the future, we're likely to change: the root and intermediate certificates from which we issue; the hash algorithms we use when signing certificates; the types of keys and key strength checks for which we are willing to sign end-entity certificates; and the ACME protocol. We will always aim to give as much advance notice as possible for such changes, though if a serious security flaw is found  ...
 
+Similarly, we're likely to change the URL of the terms of service (ToS) as we update it. Avoid hardcoding the ToS URL and instead rely on the [`Link: rel="terms-of-service"`](https://github.com/ietf-wg-acme/acme/blob/master/draft-ietf-acme-acme.md#registration) header to determine which ToS URL to submit with new registrations.
```
The spec link should probably be to https://tools.ietf.org/html/draft-ietf-acme-acme-03.

rolandshoemaker commented:
Other than the link comment this LGTM.

cpu commented:
@rolandshoemaker I changed both the URL that I added as well as the pre-existing URL in that section (It was pointed at the Github ietf-wg-acme repo as well).

@bdaehlie Any objections to merging this?
